src/libs/colors.js

```diff
@@ -23,6 +23,7 @@ const colorPalette = Utils.cond(
   [isAnvil(), { ...baseColors, primary: '#e0dd10', accent: '#035c94', light: '#f6f7f4', dark: '#012840' }],
   [isBioDataCatalyst(), { ...baseColors, primary: '#c02f42', secondary: '#1a568c', accent: '#1a568c', light: '#f4f4f6', dark: '#12385a' }],
   [isUKBiobank(), { ...baseColors, primary: '#005f6f' }],
+  [isBaseline(), { ...baseColors, primary: '#c41061', secondary: '#31164c', accent: '#4d72aa', success: '#74ae43', warning: '#f7981c', danger: '#db3214', light: '#f6f7f4', dark: '#012840' }],
```

Comment:
For the colors that are the same as the `baseColors` above, we can just omit them here.